src/lib/components/button/stories/index.stories.tsx

```diff
@@ -34,19 +34,45 @@ export const Filled: Story = {
     children: 'Label',
     disabled: false,
   },
-  play: async ({ canvasElement }) => {
+  play: async ({ canvasElement, step, args }) => {
     const canvas = within(canvasElement);
 
     const button = canvas.getByRole('button');
     const computedStyle = getComputedStyle(button);
 
     const background = computedStyle.getPropertyValue('background');
+    const color = computedStyle.getPropertyValue('color');
+    const cursor = computedStyle.getPropertyValue('cursor');
+
+    await step('onClick button test', async () => {
+      await userEvent.click(canvas.getByRole('button'));
+    });
 
-    // check if the label is being render
-    expect(button).toHaveTextContent('Label');
+    await step('Double click button test', async () => {
+      await userEvent.dblClick(canvas.getByRole('button'));
+    });
+
+    await step('Hover button test', async () => {
+      await userEvent.hover(canvas.getByRole('button'));
+    });
+
+    await step('Unhover button test', async () => {
+      await userEvent.unhover(canvas.getByRole('button'));
+    });
```

Comment:
Find one way check if this event occurr on the elements

Reply:
<img width="673" alt="Screenshot 2024-06-19 at 17 54 43" src="https://github.com/interest-protocol/interest-ui/assets/37873239/bded83c9-d196-4308-bb70-756c81f6618a">

Reply:
@KipandaJr when you validate this, please resolve the conversation